parsec/core/backend_connection/authenticated.py

```diff
@@ -59,6 +71,68 @@ def copy_exception(exception: BaseExceptionTypeVar) -> BaseExceptionTypeVar:
     return result
 
 
+# Authenticated commands types
+class DeviceCreateCallbackType(Protocol):
+    def __call__(
+        self, device_certificate: bytes, redacted_device_certificate: bytes
+    ) -> Awaitable[DeviceCreateRep]:
+        ...
```

Comment:
Is it normal to create empty `def` in python file ?

Reply:
I'll remove theses protocol once #3325 merged. They're not needed if `expose_cmds` and `expose_cmds_with_retrier` are properly typed which is done by #3325 (these functions are parts of `core.backend_connection`).